server/internal/adapter/gql/resolver_mutation_model.go

```diff
@@ -13,12 +13,13 @@ func (r *mutationResolver) CreateModel(ctx context.Context, input gqlmodel.Creat
 	if err != nil {
 		return nil, err
 	}
-
+	var m *interfaces.CreateModelParam
```

Comment:
Can you please explain why this is necessary? Seems like an unnecessary change to me.

Reply:
@rot1024 
I thought it was not a good idea to add a Public field to an already generated CreateModelParam, so I set it as a variable in advance.

```
input is defined like this
input gqlmodel.CreateModelInput
type CreateModelInput struct {
	ProjectID   ID      `json:"projectId"`
	Name        *string `json:"name"`
	Description *string `json:"description"`
	Key         *string `json:"key"`
}


input gqlmodel.UpdateModelInput
type UpdateModelInput struct {
	ModelID     ID      `json:"modelId"`
	Name        *string `json:"name"`
	Description *string `json:"description"`
	Key         *string `json:"key"`
}
```